ledger/accountdb.go

```diff
@@ -3739,7 +3797,7 @@ func updateAccountsHashRound(ctx context.Context, tx *sql.Tx, hashRound basics.R
 
 // totalAccounts returns the total number of accounts
 func totalAccounts(ctx context.Context, tx *sql.Tx) (total uint64, err error) {
-	err = tx.QueryRowContext(ctx, "SELECT count(*) FROM accountbase").Scan(&total)
+	err = tx.QueryRowContext(ctx, "SELECT count(1) FROM accountbase").Scan(&total)
```

Comment:
my understanding is that count(1) and count(*) always yield the same result, right?